tw.yaml

```
@@ -25,6 +25,17 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: 0f82e9986378ba8a01baf96f77bc76b5386ab061
 
+  # fixes mismatch between current go version and what is defined in go.work
+  - runs: |
+      go work edit -go=1.25.3
+      go work use
```

```suggestion
  # fixes mismatch between current go version and what is defined in go.work
  - runs: |
      GOVERSION=$(go version | sed -n 's/^go version go\([0-9.]*\).*/\1/p')
      go work edit -go=$GOVERSION
      go work use
```

So that in future version we don't need to manually update the go version